cloud_blobstore/__init__.py

```diff
@@ -185,6 +192,19 @@ def get_copy_token(
         """
         raise NotImplementedError()
 
+    def get_last_modified_date(
```

Comment:
I think this is safe to PR separately and land, if you have need of this API.

Reply:
Since I need both of the new functions for my other changes, I can wait to land them together in this PR